sdk/javascript/package.json

```diff
@@ -6,7 +6,9 @@
   "scripts": {
     "lint": "eslint .",
     "test": "mocha --full-trace --recursive ./test",
+    "coverage:test": "nyc --require mocha --reporter=lcov npm run test",
     "vectors": "node ./vectors/all.js --vectors $(git rev-parse --show-toplevel)/tests/vectors/${BLOCKCHAIN}/crypto --blockchain ${BLOCKCHAIN}",
+    "coverage:vectors": "nyc --require mocha --no-clean --reporter=lcov npm run vectors",
```

Comment:
if this only for jenkins, can rename to vectors:jenkins

Reply:
would run locally too so prefer to remove the `` vectors:jenkins``

Reply:
Thinking about this more would only keep the jenkins scripts if we still plan to have nightly runs.

Reply:
you want nightlies without code coverage?

Reply:
We can run code coverage at night also but it should be the same as last check-in to dev I think.  When we start running e2e then code coverage should be enabled since those dont run on every check-in.  Nightly would be more to collect test results.